panel/src/components/Forms/Field/CheckboxesField.vue

```diff
@@ -6,6 +6,38 @@
 		:input="id + '-0'"
 		:style="$attrs.style"
 	>
+		<template #counter>
+			<k-counter
+				v-if="counterOptions"
+				v-bind="counterOptions"
+				:required="required"
+				class="k-field-counter"
+			/>
+
+			<k-button-group v-if="batch" layout="collapsed">
+				<k-button
+					:disabled="value.length === 0"
+					:responsive="true"
+					icon="deselect-all"
+					size="xs"
+					variant="filled"
+					@click="deselectAll"
+				>
+					{{ $t("deselect.all") }}
+				</k-button>
+				<k-button
+					:disabled="value.length === options.length"
```

Comment:
```suggestion
					:disabled="value.length >= options.length"
```

I know not a likely scenario, but maybe if the content file has still 4 values, but in the blueprint all but 2 options are removed.